package.json

```diff
@@ -5,7 +5,7 @@
   "license": "MIT License, Copyright 2020 Adobe Systems Incorporated",
   "scripts": {
     "build": "aem-site-template-builder",
-    "deploy": "SITE_TEMPLATE_ZIP=\"$(node -p -e \"require('./package.json').name\")-$(node -p -e \"require('./package.json').version\").zip\" && curl -u admin:admin -F file=$SITE_TEMPLATE_ZIP http://localhost:4502/conf/global/site-templates.import.html"
+    "deploy": "curl -u admin:admin -F 'file=@aem-site-template-basic-latest.zip' http://localhost:4502/conf/global/site-templates.import.html"
```

Comment:
I don't think this will work as you cannot import the same file twice in AEM. 
Is there a way to detect the OS and run a specific command when OS = Windows?

Reply:
I have tested that before and double checked it right now and it works fine. Moreover previously we used to have very similar solution with common file name for deploys and that was not a problem. We extract Site Template in import servlet and we do not store the same file twice per se.

Even though zip file name is the same for different site template version they are properly displayed in the site wizzard with proper version visible.